src/styles/utilites/variables.scss

```diff
@@ -0,0 +1,9 @@
+// colors
+$light-blue: #00acdc;
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$card-hover-header: #34568b;
```

Comment:
Don't name variables with color name inside. Try to avoid using words `blue`, `white`... 
Also add `-color` to the end of the name, it will be easier to understand that this variable is for color